qemu/tests/live_backup_base.py

```diff
@@ -213,3 +228,16 @@ def verify_md5s(self):
         """
         file_names = self.params["file_names"]
         return map(self.verify_md5, file_names.split())
+
+    def verify_efficiency(self):
+        if not self.incremental_backup_time < self.full_backup_time:
+            raise exceptions.TestFail("incremental backup time %s, "
+                                      "larger than full backup time %s " %
+                                      (self.incremental_backup_time,
+                                       self.full_backup_time))
+        full_size = self.get_image_size(self.image_chain[0])
+        incre_size = self.get_image_size(self.image_chain[1])
+        if incre_size > full_size:
```

Comment:
@pezhang Thanks, the test step is incremental backup after guest run for time.sleep(120)  after full backup,  this case incremental backup will record very limited data, it should save most space compare to full backup, that is why incremental backup exists, and this test case is suggested by qemu devel.